tests/integration/tests/test_util/util.py

```diff
@@ -247,3 +247,35 @@ def get_default_ip(instance: harness.Instance):
         ["ip", "-o", "-4", "route", "show", "to", "default"], capture_output=True
     )
     return p.stdout.decode().split(" ")[8]
+
+
+def configure_lxd_profile(
+    profile_name: str, profile_config: str, template_overwrites: Dict[str, str] = {}
+):
```

Comment:
i'd make this a `def _configure_profile` on the LXD harness. in the future, we might want to configure how we talk to lxd, and having this as part of LXDHarness is simpler (keeps all LXD code close)